docs/schema.md

```diff
@@ -2,23 +2,39 @@
 
 You need a `schema.yaml` file when you [build your deployer](building-deployer.md).
 
-You use the schema file to declare parameter values that users must provide
-when they deploy your application to their clusters. The information that users
-provide is available to your application when it is deployed.
-
-For example, the schema includes the name of the application instance, the
-Kubernetes namespace, service accounts, and so on. Your schema definition is
-used to create the form that users see when they deploy the application from
-the GCP Console.
-
-The format of `schema.yaml` follows a subset of JSON schema specifications.
-
-In your deployer image, you must add the schema file to `/data/schema.yaml`.
-
-This is a basic example of a schema.yaml file:
+The schema file provides the following:
+- Information about the current release, such as its release version, release
+  notes, etc.
+- A declaration of all images used by the app. These images must be
```

Comment:
Can you add __and parameterization__ to this paragraph. The explanation goes something like this:

A __declaration__ and __parameterization__ of __all__ images used by the app.
- These images must be explicitly declared...
- The manifests or templates must accept the images as __parameters__ because the final locations of the images are only known at deployment time. The image parameters are similar to the `properties` declared in the schema.